### DISCUSSION TITLE
Does neoclide means neovim as client ide?

### DISCUSSION THREAD

**Discussion by sisrfeng** [neoclide/coc.nvim#4149] [General]
neoclide = **neo**vim   as **cl**ient (tui) **ide** ?

btw, neovide may means **neov**im as **ide** (though it is actually just a GUI front-end)

**Comment by chemzqm** [comment]
Neovim as backend commandline ide, maybe

**Comment by oblitum** [comment]
neo doesn't necessarily needs to be tied to neovim, it's a prefix used for many things, specially after the Matrix movies, it's why neovim is called neovim itself.

So, neoclide could also, maybe, mean, Neo CLI  IDE. Where the two I's get merged and not being specifically tied to neovim, given it works with vim.
